packages/common/src/content.ts

```diff
@@ -66,7 +66,7 @@ const cache: { [key: string]: string } = {};
 export function getCategory(itemType: string = ""): string {
   /* tslint:disable no-console */
   console.warn(
-    "DEPRECATED: Use getCollection() instead. getCategory will be removed at v9.0.0"
+    "DEPRECATED: Use getCollection() instead. getCategory will be removed at v10.0.0"
```

Comment:
Simply updating log statements for things we flagged for removal in v9.0.0 but didn't get the axe when that release was published.